docs/integrations.md

```diff
@@ -0,0 +1,16 @@
+Atomist receives events from and performs operations on many systems,
+including version control, CI systems, Slack, and more.
```

Comment:
If CI has systems, why wouldn't version control? I'd recommend being consistent.

Reply:
I cut & pasted this from the original intro page